lib/cdo/pegasus/graphics.rb

```diff
@@ -1,6 +1,12 @@
 require 'rmagick'
 
-def load_manipulated_image(path,mode,width,height=nil)
+MAX_DIMENSION = 2880
+
+def load_manipulated_image(path, mode, width, height = nil)
+
+  width = [MAX_DIMENSION, width].min
```

Comment:
We're pretty sure that we will not legitimately ask for an image > 2880, so we don't care about keeping the aspect ratio if one or more of the dimensions is >?